tests/conftest.py

```diff
@@ -139,6 +139,16 @@ def mint():
 
     server = UvicornServer(config=config)
     server.start()
-    time.sleep(1)
+
+    # Wait until the server has bound to the localhost socket. Max out after 10s.
+    tries = 0
+    while tries < 100:
+        try:
+            httpx.get(settings.mint_url)
+            break
+        except Exception:
```

Comment:
We should specify which exception we are expecting:
```suggestion
        except httpx.ConnectError:
```

Reply:
Good idea, applied